app/controllers/admin/dashboard_controller.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
 class Admin::DashboardController < Admin::AdminController
-  def index; end
-end
+  def index
+    @articles = Article.where.not(status: 0)
```

Comment:
```
@articles = !Article.pending
```

Reply:
```
Article.where.not(status: :pending)
```

Reply:
```
Article.not.pending
```
This will probably not fly

Reply:
the only one that works is this one 
```
Article.where.not(status: :pending)
```
so will change to that